src/app/home/home.component.ts

```diff
@@ -56,6 +59,7 @@ export class HomeComponent implements OnInit, AfterViewInit {
 
   ngOnInit() {
     this.user$ = this.userQuery.user$;
+    this.userQuery.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(user => this.user = user);
```

Comment:
No need to subscribe just for the username.
Make username another observable or just (user$ | async).username
Could've extend `Base` to avoid things like missing onDestroy